nservicebus/pipeline/satellites.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrating Satellites
+summary: Extension point for handling messages
+tags: 
+related:
+ - nservicebus/pipeline/customizing-v6
+ - nservicebus/pipeline/features
+redirects:
+---
+
+In NServiceBus Versions 5 and below, Satellites were extension points to handle messages that arrive in different queues other than the endpoint's main queue. This extension point came in handy when the robustness of a separate queue was needed without having to create and setup a brand new Endpoint and configure any needed message mappings.
```

Comment:
Should we use past tense here? This is still the way to do things in v5. Also, are Satellites still an abstraction? This opening paragraph makes it sounds like they are gone but then the rest of the document explains how to set one up.

At a high level you could restructure this as:

- Satellites are parts of an Endpoint that handle messages sent to a queue other than the main input queue. 
- In Version 5 and below, these were provided with `ISatellite` and `IAdvancedSatellite`. 
- In Version 6 and above, these interfaces are deprecated.
- In Version 6 and above, you create a Satellite by setting up a new pipeline from within a feature. Link to features.
- NOTE: In order for the satellite to process messages, the feature that creates it must be enabled. Link to how to do that.
- Details on how to create the pipeline. 
- Sample on how to terminate the pipeline. *I would guess that if you want to go really advanced you could create stage connector in here and start building a very complex pipeline. Given that most people won't do that, it may not be worth bringing that up*
- Advanced Satellites capabilities are still available.

It's pretty close to the way you have it already.